### PR TITLE
Fixes and design improvments

### PR DESCRIPTION
**Pull request by sBouzols**

### CODE REVIEW

File: src/components/app.js [hunk 1]
```diff
@@ -75,6 +75,13 @@ const App = () => {
 
     const location = useLocation();
 
+    //remove the default contextMenu
```
Comment (ChamseddineBhd) [inline src/components/app.js]: redundant L 110 in the same file

**Review by ChamseddineBhd — CHANGES_REQUESTED** [review]

File: src/components/directory-content.js [hunk 2]
```diff
@@ -53,9 +53,22 @@ const useStyles = makeStyles((theme) => ({
...
         flex: 1,
         height: '48px',
-        cursor: 'initial',
         padding: DEFAULT_CELL_PADDING,
     },
+    chip: {
+        cursor: 'pointer',
+    },
+    icon: {
+        display: 'flex',
```
Comment (ChamseddineBhd) [inline src/components/directory-content.js]: why flex?

File: src/components/directory-content.js [hunk 3]
```diff
@@ -287,41 +300,39 @@ const DirectoryContent = () => {
...
-                            <FormattedMessage id="creationInProgress" />
-                        </>
-                    )}
-                </div>
+
+                {childrenMetadata[elementUuid] ? (
+                    <div>{childrenMetadata[elementUuid].name}</div>
+                ) : (
+                    <>
+                        {elementName}{' '}
```
Comment (ChamseddineBhd) [inline src/components/directory-content.js]: {' '} ?

File: src/components/directory-tree-view.js [hunk 4]
```diff
@@ -562,10 +617,14 @@ const DirectoryTreeView = ({ rootDirectory, updateRootDirectories }) => {
     return (
         <>
             <TreeView
-                className={classes.root}
-                defaultCollapseIcon={<ExpandMoreIcon />}
+                className={classes.treeViewRoot}
+                defaultCollapseIcon={
+                    <ExpandMoreIcon className={classes.icon} />
+                }
                 defaultExpanded={['root']}
```
Comment (ChamseddineBhd) [inline src/components/directory-tree-view.js]: defaultExpanded to be deleted ?

**Review by ChamseddineBhd — CHANGES_REQUESTED** [review]

File: src/components/directory-content.js (hunk 3 above)
Comment (sBouzols) [reply src/components/directory-content.js]: It adds a whitespace between `elementName` and the `creationInProgress` message

**Review by ChamseddineBhd — APPROVED** [review]

**Pull request merged**